apple/RNSVGRenderable.mm

```diff
@@ -250,7 +250,10 @@ - (void)renderTo:(CGContextRef)context rect:(CGRect)rect
 
   [self beginTransparencyLayer:context];
 
-  if (self.mask || self.filter) {
+  RNSVGFilter *filterNode = self.filter ? (RNSVGFilter *)[self.svgView getDefinedFilter:self.filter] : nil;
```

Comment:
Why do we need casts here? Shouldn't the method return type be `RNSVGFilter`?

Reply:
Fixed in [ea60684](https://github.com/software-mansion/react-native-svg/pull/2360/commits/ea60684b087e54a4541d1c550f371f77de6abe02)